packages/sdk/src/otlp-exporter.ts

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2023 SolarWinds Worldwide, LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { type ExportResult } from "@opentelemetry/core"
+import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-grpc"
```

Comment:
Has grpc export to the custom otel-collector been working well? Asking for myself as I've only done http export to it so far.

Reply:
Mentioned during standup but will also reply here for future reference, yes grpc has been working fine for me